com.wso2telco.dep.common.mediation/src/main/java/com/wso2telco/dep/common/mediation/spendlimit/CustomException.java

```diff
@@ -0,0 +1,41 @@
+package com.wso2telco.dep.common.mediation.spendlimit;
+
+
+public class CustomException extends RuntimeException {
```

Comment:
This entire class needs to be removed. Exceptions are handled differently in synapse